kuma/landing/views.py

```diff
@@ -99,6 +99,7 @@ def promote_buttons(request):
 Disallow: /*move-requested
 Disallow: /*preview-wiki-content
 Disallow: /*profiles*/edit
+Disallow: /*/search
```

Comment:
This will block page like:

* https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/search
* https://developer.mozilla.org/en-US/docs/Web/Events/search
* https://developer.mozilla.org/en-US/docs/Web/API/URL/search

Reply:
Good catch!

Reply:
I think you're right @jwhitlock, the robots.txt language is too limited to single-out the search endpoint. I think we have to settle for updating the `meta` tag for the search page from what we have currently
```
<meta name="robots" content="noindex, follow">
```
to this
```
<meta name="robots" content="noindex, nofollow">
```
to get the same/similar effect.